Signal/src/ViewControllers/DebugUI/DebugUISessionState.m

```diff
@@ -13,10 +13,19 @@
 
 @implementation DebugUISessionState
 
-+ (OWSTableSection *)sectionForContactThread:(TSContactThread *)contactThread
```

Comment:
Why did you get rid of this? It was intended to indicate that this could only be used for ContactThreads.

Most of these actions don't make sense for group threads. 

e.g. line 54

```
+                                           NSString *recipientId = [thread contactIdentifier];
```